determined-ai/determined
# fix: links for Recent Submissions on Dashboard

## Ticket: [ET-604](https://hpe-aiatscale.atlassian.net/browse/ET-604)




## Description

On dashboard with Flat Runs feature flag on, Recent Submissions should link to Search Details page instead of Experiment Details page

## Test Plan

- Enable the Flat runs feature
  - go to the dashboard page
  - click on any Recent Submission item
  - it should navigate to the `/searches/<id>`

- Disable the Flat runs feature
  - go to the dashboard page
  - click on any Recent Submission item
  - it should navigate to the `/experiments/<id>`




## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ET-252]: https://hpe-aiatscale.atlassian.net/browse/ET-252?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ET-309]: https://hpe-aiatscale.atlassian.net/browse/ET-309?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ET-604]: https://hpe-aiatscale.atlassian.net/browse/ET-604?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
